bitnami/postgresql/templates/tls-secrets.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if (include "postgresql.createTlsSecret" . )}}
+{{- $ca := genCA "postgresql-internal-ca" 365 }}
```

Comment:
Just curious why the `-internal-` affix?

Reply:
I added this affix to say that it is a certificate to use internally (not in production) but looking the decision with perspective, it is not the best name. I changed it.